components/admin-panel/sections/ReceiptTemplateForm.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Flex } from '../../Grid';
+import PreviewModal from '../../PreviewModal';
+import StyledButton from '../../StyledButton';
+import StyledInput from '../../StyledInput';
+import StyledTextarea from '../../StyledTextarea';
+import { Label, P, Span } from '../../Text';
+
+const ReceiptTemplateForm = ({ defaultTemplate, value, onChange, placeholders }) => {
+  const [showPreview, setShowPreview] = React.useState(false);
+
+  return (
+    <React.Fragment>
+      <Label htmlFor="receipt-title" color="black.800" fontSize="16px" fontWeight={700} lineHeight="24px">
+        <FormattedMessage defaultMessage="Receipt title" />
+      </Label>
+      <StyledInput
+        id="receipt-title"
```

Comment:
:no_entry_sign: There's going to be a collision on the fields' IDs, we'll have multiple inputs with the same one. Suggested fix: accept a `template` (string) prop and do something like:

 ```suggestion
         id={`receipt-title-${template}`}
```